lib/jquery-rjs/rendering.rb

```diff
@@ -1,13 +1,21 @@
 require 'action_view/helpers/rendering_helper'
 
-ActionView::Helpers::RenderingHelper.module_eval do
-  def render_with_update(options = {}, locals = {}, &block)
-    if options == :update
-      update_page(&block)
-    else
-      render_without_update(options, locals, &block)
-    end
+module JqueryRjs::RenderingHelper
+  method_name = "render#{'_with_update' if RUBY_VERSION < '2'}"
```

Comment:
Is there a way to do this that doesn't require updating this method?  By that, I mean can we keep this method the same, and just include a wrapper method for `render` that includes invoking this method.  Something like:

```ruby
module JqueryRjs::RenderingHelper
  def render_with_update(options = {}, locals = {}, &block)
    if options == :update
      update_page(&block)
    else
      render_without_update(options, locals, &block)
    end

  if RUBY_VERSION >= '2'
    def render_without_update(*args)
      render(*args)
    end
  end
end
```

Not sure if this would work, but I am going off what you did.

Something to keep in mind is you are also changing the functionality of the function call with this newly defined method.  In `RUBY_VERSION < '2'`, it goes from the deleted code to this:

```ruby
def render_with_update(options = {}, locals = {}, &block)
  if options == :update
    update_page(&block)
  end
  render_without_update(*args)
end
```

So `render_without_update` is always called instead of when `options != :update`.  Is this expected?